lib/cadet/assessments/assessments.ex

```diff
@@ -57,11 +55,11 @@ defmodule Cadet.Assessments do
   end
 
   def all_open_assessments(assessment_type) do
-    now = Timex.now()
-
-    assessment_with_type = Repo.all(from(a in Assessment, where: a.type == ^assessment_type))
-    # TODO: Refactor to be done on SQL instead of in-memory
-    Enum.filter(assessment_with_type, &(&1.is_published and Timex.before?(&1.open_at, now)))
+    Assessment
```

Comment:
Same thing w.r.t. `max_xp` field

Reply:
i'm thinking of removing this method since assessments_controller is implemented and doesn't use it

Reply:
Sure, I don't think anyone's gonna miss it.